### PR TITLE
IA restructure

### PR DESCRIPTION
**Pull request by SueSmith**
@claireriley ok 😅 whenever you're ready lol, it's also here https://postman-learning-test.netlify.app/

There are a few pages where it looks like new content but has just been condensed i.e. I've copied content from one page to another, esp these (so no need to do a full check on the content on these as long as nothing is obviously broken..):
* Creating workspaces
* Using and managing workspaces
* Navigating Postman
* API submission guidelines
* Creating run in postman buttons
* Building workflows
* Troubleshooting api requests

Anything that looks like a new page and isn't on that list is either genuinely new or signficantly changed so if you could prioritise checking those over that would be great (e.g. the /developer resources section, the new pages in /getting-started etc). If you could have a decent look at the LC homepage and /docs intro page too that would be magic.. 🙏 

A lot of the edits are just link updates and ordering.. I had also started removing the word "app" in some places before I knew we weren't updating for artemis yet.. 🙃 

Need to check with @ch264 how to remove `references.jsx` because whenever I do it breaks the style for some reason.. have removed other pages with no trouble, nbd for now since it's not linked anywhere

### CODE REVIEW

**Comment by ch264** [comment]
@SueSmith @claireriley I updated the snapshots and fixed the linter errors. This looks awesome! 🚀

**Comment by SueSmith** [comment]
Thanks @ch264! 🙌

File: redirects.json [hunk 1]
```diff
@@ -1,622 +1,1226 @@
...
+		"from": "/docs/postman/api_documentation/intro_to_api_documentation/",
+		"to": "/docs/publishing-your-api/documenting-your-api/"
+	},
+	{
+		"from": "/docs/postman/api_documentation/publishing_public_docs/",
+		"to": "/docs/publishing-your-api/publishing-your-docs/"
+	},
+	{
+		"from": "/docs/postman/collaboration/adding-private-network/",
+		"to": "/docs/publishing-your-api/add-api-network/"
```
Comment (claireriley) [inline redirects.json:428]: In your spreadsheet this one says it should go to /docs/collaborating-in-postman/adding-private-network/

File: redirects.json [hunk 2]
```diff
@@ -1,622 +1,1226 @@
...
+	{
+		"from": "/docs/postman/collections/version-control-for-collections/",
+		"to": "/docs/collaborating-in-postman/version-control-for-collections/"
+	},
+	{
+		"from": "/docs/postman/collections/version_control/",
+		"to": "/docs/collaborating-in-postman/version-control-for-collections/"
+	},
+	{
+		"from": "/docs/postman/collections/working-with-openAPI/",
```
Comment (claireriley) [inline redirects.json:555]: This is the page I want to save

File: src/pages/docs/developer/collection-conversion.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    name: "Continuous Integration"
+    url: "/docs/running-collections/using-newman-cli/continuous-integration/"
+
+warning: false
+
+---
+
+You can convert from a variety of API formats into Postman collections. If you have API requests defined as OpenAPI, Swagger, RAML, or GraphQL schema formats, you can convert these to Postman collections that you can import into the app or use with other Postman utilities such as the Newman CLI.
+
+The following open source projects are hosted on GitHub:
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/developer/collection-conversion.md:24]: so this section is more for promoting the open source libraries etc, plan is to build the info out later although it's just kind of a stub atm..

**Comment by SueSmith** [comment]
Thanks @claireriley so many mistakes 😭 - I think I've addressed everything..

**Comment by SueSmith** [comment]
🤦‍♀️ thanks @claireriley - sorted

**Pull request merged**